docs/source/sci_api_tutorial/armer.rst

```diff
@@ -2,21 +2,280 @@
+
+.. dropdown:: :fa:`eye,mr-1` Click to learn more about `Dynamic monitoring`
+    
+    The capability of dynamically monitoring the job completion status is vital to high-throughput modeling workflow because the workflow involves multiple different types of simulation subtasks that must be sequentially operated.
+    
+    Two methods have been implemented to achieve dynamic monitoring, they are: ``wait_to_end()`` and ``wait_to_array_end()`` methods. The ``wait_to_end()`` method checks the status of a job in the job queue within a certain period of time (i.e., every 30 s) and exits upon the detection of messages that indicate job completion, error, or cancellation. The ``wait_to_array_end()`` method takes multiple job objects and submits them in one job array. Similarly, this method also monitors the status of all jobs in the array regularly and dynamically appends new jobs to the array up to the maximal capacity (i.e., array size).
+
+.. dropdown:: :fa:`eye,mr-1` Click to learn more about the arguments in Dynamic monitoring
+        
+    ``period``
+        the time cycle for update job state change (Unit: s)
+    
+    The following arguments are array submission only:
+
+    ``jobs``
+        a list of ClusterJob object to be execute
+        
+    ``array_size``
+        
+        how many jobs are allowed to submit simultaneously. 
+
+        (e.g. 5 for 100 jobs means run 20 groups. All groups will be submitted and 
+        in each group, submit the next job only after the previous one finishes.)
+        
+    ``sub_dir``
+        (default: self.sub_dir)
+
+        submission directory for all jobs in the array. 
+        
+        Overwrite existing self.sub_dir in the job obj
+        
+        * you can set the self value during config_job to make each job different
+    
+    ``sub_scirpt_path`` 
+        (default: self.sub_script_path)
+        
+        path of the submission script. Overwrite existing self.sub_script_path in the job obj
+        
+        * you can set the self value during config_job to make each job different
+
+
+
+cluster_job_config
 ==============================================
-| This section describe a soft-enforced standard format of 
-  the configuration dictionary used as input of all the 
-  Science APIs.
-| For example: 
-| ``quantum.single_point()`` needs 
-  to know what resource/account it needs to use when running
-  commands through ARMer. We have a ``cluster_job_config``
-  argument in this function as a dictionary obtain this information.
-  The format of this dictionary is soft-enforced.
-
-TODO
-  
+
+However, none of the function will be directly access by user in normal EnzyHTP workflows. They are used by developers in APIs of EnzyHTP. But all of the APIs that uses ARMer will expose an input that allow users to config the resource.
```

Comment:
Put the words about "only cluster_job_config is important" at the end of the previous section and Bold them